code/datums/abilities/flock/flockmind.dm

```diff
@@ -343,9 +343,16 @@
 	if(!istype(T, /turf/simulated/floor/feather))
 		boutput(holder.owner, "<span class='alert'>You aren't above a flocktile.</span>")//todo maybe make this flock themed?
 		return 1
+	if(locate(/obj/flock_structure/ghost) in T)
+		boutput(holder.owner, "<span class='alert'>A tealprint has already been scheduled here!</span>")
+		return 1
 	if(locate(/obj/flock_structure) in T)
 		boutput(holder.owner, "<span class='alert'>There is already a flock structure on this flocktile!</span>")
 		return 1
+	for (var/atom/O in T.contents)
+		if (O.density && !isflock(O))
+			boutput(holder.owner, "<span class='alert'>A tealprint cannot be scheduled with [O] on the same tile!</span>")
```

Comment:
```suggestion
			boutput(holder.owner, "<span class='alert'>A tealprint cannot be scheduled with \a [O] on the same tile!</span>")
```

Reply:
> 

\a or \the? Some objects already have "the" in their name I think, so maybe \the would be a little better for consistency?